### PR TITLE
And I just noticed...

### PR DESCRIPTION
**Pull request by robske110**
...we really did never give @PocketMine credit :o

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,14 +6,14 @@
...
-# We present you: ClearSky-Sunrise (PHP5)
+# We present you: ClearSky-Sunrise ~ A [PM](https://github.com/PocketMine/PocketMine-MP) fork (PHP5) 
 ClearSky is an ultra fast Minecraft: Pocket Edition server software with clean feels and stable features. It was initially designed for production servers.
 
 **TravisCI Build Status** [![TravisCI](https://travis-ci.org/ClearSkyTeam/ClearSky.svg?branch=0.14.0)](https://travis-ci.org/ClearSkyTeam/ClearSky "TravisCI Build Status")
 **Jenkins Build Status** [![Build Status](http://jenkins.clearskyteam.org/buildStatus/icon?job=ClearSky)](http://jenkins.clearskyteam.org/job/ClearSky/ "Jenkins Build Status")
 
 
- - We may take code from other GPL licenced projects, but most features are fully-rewritten for ClearSky.
+ - We may take code from other GPL licenced projects, but very much is fully-rewritten for ClearSky.
```
Comment (inxomnyaa) [inline README.md]: Dat englisch is beste

Comment (robske110) [inline README.md]: Stimmt. Mist

File: README.md [hunk 2]
```diff
@@ -55,12 +55,13 @@ Note¹ - Please set 'Outdated Server' message to 'false' in the CustomAlert plug
 Note² - Please set 'Full server' message to 'false' in the CustomAlert plugin or any related plugin if you encounter *Full server*
 
 ## For Developers
-This is a clean, high quality code base. Developing/modifying this project is easy.<br>
+This is a clean, high quality code base, because we are trying to keep PocketMine's great code quality.
```
Comment (inxomnyaa) [inline README.md:58]: haha, great. best joke

File: README.md [hunk 3]
```diff
@@ -55,12 +55,13 @@ Note¹ - Please set 'Outdated Server' message to 'false' in the CustomAlert plug
...
+Developing/modifying this project is easy.<br>
```
Comment (inxomnyaa) [inline README.md:59]: no1 has to care

File: README.md [hunk 4]
```diff
@@ -55,12 +55,13 @@ Note¹ - Please set 'Outdated Server' message to 'false' in the CustomAlert plug
...
 We are still rewriting the base to make sure ClearSky has the best developing feel.<br>
 ClearSky is not just for a CLEAN feel for users - it's also for developers!<br>
 
 # <a name="SPA"></a>Español
-# Te presentamos: ClearSky-Sunrise (PHP5)
+# Te presentamos: ClearSky-Sunrise ~ A [PM](https://github.com/PocketMine/PocketMine-MP) fork (PHP5)
```
Comment (inxomnyaa) [inline README.md]: and.. the other languages? xD

File: README.md (hunk 2 above)
Comment (robske110) [inline README.md:58]: But we're trying to do this...

File: README.md (hunk 3 above)
Comment (robske110) [inline README.md:59]: That line was there already -_-

File: README.md (hunk 4 above)
Comment (ghost) [inline README.md]: @robske110 You are a great person because you don't steal code from other person

**Comment by ghost** [comment]
@robske110 I could help you to translate to other languages

**Comment by robske110** [comment]
Ok, go to the branch and make an PR in the branch of this PR

File: README.md (hunk 3 above)
Comment (inxomnyaa) [inline README.md:59]: woops xd

File: README.md (hunk 2 above)
Comment (inxomnyaa) [inline README.md:58]: we are fixing pocketmine. Blame the PM "code quality"

File: README.md [hunk 5]
```diff
@@ -109,10 +110,10 @@ Nota¹ - Porfavor poner el mensaje 'Outdated Server' a 'false' en el plugin Cust
 Nota² -Porfavor poner el mensaje 'Full Server' a 'false' en el plugin CustomAlert, o algun plugin parecido si se encuentra *FullServer* al entrar.
   		  
 ## Para desarrolladores
-ClearSky es un software limpio con codigo de alta calidad.<br>		 
+ClearSky es un software limpio, porque tratamos de conservar rl codigo de alta calidad de PocketMine.<br>
```
Comment (ghost) [inline README.md]: My fault please change that rl to el

**Comment by robske110** [comment]
Finished.

**Comment by ghost** [comment]
@thebigsmileXD "we are fixing pocketmine. Blame the PM "code quality"" that was great

**Comment by robske110** [comment]
Nope, actually it was a lie.
No, it was not as he just blamed it
But you shouldn't blame it, because it was realy good...
And we're adding new features, we are not fixing it

**Comment by inxomnyaa** [comment]
There you said it. "Was"

**Comment by robske110** [comment]
The code quality is still very good, as there have been no changes for ∞^∞ (∞ to the power of ∞) decades.

**Comment by inxomnyaa** [comment]
:laughing: Nooo.. look at the mess since php7. It looks weirder than Bukkit (compiled bukkit xD)

**Comment by Aericio** [comment]
Is php7 worse than php5 then?

**Comment by ghost** [comment]
@thebigsmileXD I mean, what you said was the right thing.
Today PocketMine's API sucks

**Comment by Aericio** [comment]
@SkygridRex  good to know.

**Pull request merged**